packages/components/src/templates/forms.stories.ts

```diff
@@ -0,0 +1,70 @@
+import '../solid-components';
+import { html } from 'lit-html';
+
+/**
+ * ```
+ * ```
+ */
+export default {
+  tags: ['!dev'],
+  title: 'Templates/Forms',
+  parameters: {
+    chromatic: { disableSnapshot: true },
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/design/YDktJcseQIIQbsuCpoKS4V/Component-Docs?node-id=3060-14503&t=LHCyCYMjDV22qemZ-4'
+    }
+  }
+};
+
+export const loginForm = {
+  render: () => html`
+    <section class="sd-container">
+      <form>
+        <h3 class="sd-headline sd-headline--size-3xl mb-8">Login for UnionFondsOnline users</h3>
+        <sd-input type="text" label="Access number" spellcheck class="mb-4"></sd-input>
+        <sd-input type="password" label="Access number" password-toggle spellcheck class="mb-4"></sd-input>
+        <sd-checkbox>Stay logged in on this computer</sd-checkbox>
+        <div class="justify-between md:items-center md:flex-row mt-8 flex flex-col">
+          <sd-link href="#" class="mb-4 md:mb-0">Forgot password</sd-link>
```

Comment:
```suggestion
          <sd-link href="javascript:void(0)" class="mb-4 md:mb-0">Forgot password</sd-link>
```

Could work to prevent reloading. Do for all links here.

Reply:
Updated